package.json

```diff
@@ -48,7 +48,7 @@
 		"@gov.au/pancake": "^1.1.0",
 		"@gov.au/pancake-js": "^1.0.9",
 		"@gov.au/skip-link": "^0.3.0",
-		"cuttlebelle": "^1.0.0-alpha.41",
+		"cuttlebelle": "^1.0.0-alpha.45",
```

Comment:
why the update @joolswood ?
We are on 55 now so may be worth going all the way?